openwisp_monitoring/device/tasks.py

```diff
@@ -61,6 +61,11 @@ def save_wifi_clients_and_sessions(device_data, device_pk):
             update_fields = []
             for field in _WIFICLIENT_FIELDS:
                 if getattr(client_obj, field) != client.get(field):
+                    # The wifi 6 client information (he) is optional
+                    # so when it is not available, we should make
+                    # sure its default value remains unchanged ie. False
+                    if field == 'he' and field not in client:
+                        continue
```

Comment:
here I disagree, if we don't have information about HE, we should make sure it's value is "None", because False means we are sure is not supported.